content/guides/analog-discovery/_index.md

```diff
@@ -0,0 +1,886 @@
+---
+title: 'Interacting with projects and making measurements with the Analog Discovery'
+linkTitle: Testing Projects with the Analog Discovery 
+description: 'Using the Digilent Analog Discovery 3 to control and probe projects'
+weight: 60
+---
+
+
+This is an example of using the [Digilent Analog Discovery 3](https://digilent.com/reference/test-and-measurement/analog-discovery-3/start) multi-function test and measurement device to characterize a project on the [Tiny Tapeout 05 ASIC](/runs/tt05/), in particular [#271: Analog Testmacro](https://tinytapeout.com/runs/tt05/tt_um_tt05_analog_test), which is a test analog circuit included on TT05 containing both a ring oscillator and a 3-bit DAC.
+
+
+There is a sample *WaveForms* workspace that is ready to use that can 
+   * talk to the demoboard REPL over serial to setup the correct project;
+   * drive the demoboard inputs to dynamically control the project;
+   * monitor and measure the ring oscillator and DAC outputs; and
+   * graph the results and output CSV files with data.
+
+
+The purpose here is to walk-through the entire process while introducing various features of the AD3 and some of the ways to use it.  If you just want the bullet point list of steps without any details, skip to the end of this page.
```

Comment:
the bullet point list is not at the end of the page, maybe you re-organised things?

Reply:
I did.  Ok, that bug report for the workspace is lacking but I'll go over each of these and amend the PR.